tests/test_axis_internal.py

```diff
@@ -39,60 +39,28 @@ def test_axis_regular_uoflow(axtype, function):
 def test_axis_regular_extents():
     ax = bh.axis.regular(10, 0, 1)
     assert 12 == ax.extent
-    assert 11 == len(ax.edges())
-    assert 13 == len(ax.edges(True))
-    assert 10 == len(ax.centers())
+    assert 10 == len(ax.centers)
     assert ax.options == bh.axis.options(underflow=True, overflow=True)
 
     ax = bh.axis.regular(10, 0, 1, overflow=False)
     assert 11 == ax.extent
-    assert 11 == len(ax.edges())
-    assert 12 == len(ax.edges(True))
-    assert 10 == len(ax.centers())
+    assert 10 == len(ax.centers)
     assert ax.options == bh.axis.options(underflow=True)
 
     ax = bh.axis.regular(10, 0, 1, underflow=False)
     assert 11 == ax.extent
-    assert 11 == len(ax.edges())
-    assert 12 == len(ax.edges(True))
-    assert 10 == len(ax.centers())
+    assert 10 == len(ax.centers)
     assert ax.options == bh.axis.options(overflow=True)
 
     ax = bh.axis.regular(10, 0, 1, flow=False)
     assert 10 == ax.extent
-    assert 11 == len(ax.edges())
```

Comment:
Can't we still check `ax.edges` here?

Reply:
edges was removed.

Reply:
I would prefer if the only way to get numpy.histogram like edges is via histogram.to_numpy

Reply:
histogram.to_numpy will get much more thorough testing, naturally, to cover all these cases.

Reply:
Numpy like edges could be useful for plotting and other uses that are not just trying to get a numpy like return out. Numpy itself added a way to get the edges only a couple of versions ago. This seems useful to have. https://docs.scipy.org/doc/numpy/reference/generated/numpy.histogram_bin_edges.html#numpy.histogram_bin_edges 

(This is fine for now, just something to think about)

Reply:
I am ok with adding edges as a property without `flow=True`. `histogram.to_numpy` should be only way to get edges with `flow=True`, because `to_numpy` guarantees that the shape of the counter matrix matches the edges shape.

Reply:
I think that is okay, especially for axis. Flow is a bit of a special case for edges.